src/waltz/quic/fd_quic.c

```diff
@@ -2828,6 +2833,12 @@ fd_quic_service( fd_quic_t * quic ) {
   cnt += fd_quic_svc_poll_tail( quic, FD_QUIC_SVC_INSTANT, now );
   cnt += fd_quic_svc_poll_head( quic, FD_QUIC_SVC_ACK_TX,  now );
   cnt += fd_quic_svc_poll_head( quic, FD_QUIC_SVC_WAIT,    now );
+
+  ulong after = fd_quic_now( quic );
+  ulong delta = after - now;
+
+  fd_histf_sample( quic->metrics.service_duration, delta );
```

Comment:
These histograms are in ticks (`fd_tickcount()`) not nanos for performance. The metric till will convert them to nanos for display.